### PR TITLE
fix(native-player): handle progressive sources

### PR DESCRIPTION
**Pull request by yairans**

### CODE REVIEW

File: src/utils/resolution.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        if (ratioDiff < minRatioDiff) {
+          minRatioDiff = ratioDiff;
+          mostSuitableWidthAndRatioTracks = [track];
+        } else if (ratioDiff === minRatioDiff) {
+          mostSuitableWidthAndRatioTracks.push(track);
+        }
+      }
+    }
+    let maxBandwidth = 0;
+    for (let track of mostSuitableWidthAndRatioTracks) { // select the top bitrate from the ratio filter results
```
Comment (itaykinnrot) [inline src/utils/resolution.js]: @yairans  we should consider preferred bitrate as an external input no? @OrenMe

Comment (OrenMe) [inline src/utils/resolution.js]: Yes, but all ABR section is not implemented yet, will be done as part of another PR that will handle it over all engines and adapters(native/hls/dash)
